content/en/user-manual/billing/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Billing
+template: usermanual-page.tmpl.html
+position: 16
+---
+
+## Auto-renewal of Subscriptions
+Once you subscribe to the Personal or Organization plan, payment will automatically be taken on a monthly basis, on the day of the month when you first subscribed.
+
+## Invoices
+Invoices for monthly subscriptions are emailed to the owner of the subscribed account on the monthly billing date. Invoices are not available to download on the website itself. If invoices are not reaching your inbox, please check your spam folder. If you are missing invoices from your records, you may request for them to be resent by emailing sales@playcanvas.com.
```

Comment:
May want to make the email a mailto: link

Reply:
Fixed!